code/modules/antagonists/changeling/powers/absorb.dm

```diff
@@ -93,7 +100,9 @@
 	var/datum/action/changeling/evolution_menu/E = locate() in user.actions
 	SStgui.update_uis(E)
 
+	target.deathgasp_on_death = FALSE
 	target.death(FALSE)
+	target.deathgasp_on_death = initial(target.deathgasp_on_death)
```

Comment:
Nothing else changes this, but I am still somewhat skeptical of this. I personally think the method I suggested is better, but this is technically fine.

Reply:
I think it's hella sus this is even a var I'll be real